beacon-chain/powchain/engine_client.go

```diff
@@ -33,6 +33,10 @@ const (
 	ExecutionBlockByHashMethod = "eth_getBlockByHash"
 	// ExecutionBlockByNumberMethod request string for JSON-RPC.
 	ExecutionBlockByNumberMethod = "eth_getBlockByNumber"
+	// Defines the seconds to wait before timing out engine endpoints with block execution semantics (newPayload, forkchoiceUpdated).
+	payloadAndForkchoiceUpdatedTimeout = 8 * time.Second
+	// Defines the seconds before timing out engine endpoints with non-block execution semantics.
+	defaultTimeout = time.Second
```

Comment:
can we call this defaultEngineTimeout? otherwise seems like a default for all things in the powchain service